src/Serialization/upgrades/main.jl

```diff
@@ -0,0 +1,29 @@
+# Upgrade scripts contain code to upgrade files that were serialized with an older
+# format to the proceeding file format version. Thus to upgrade a file to
```

Comment:
"proceeding" ? Do you mean "next"? (That would be clearer to me as a non-native speaker)